core/tests/protractor_desktop/editorFeatures.js

```diff
@@ -737,34 +738,28 @@ describe('Issues visualization', function() {
     libraryPage.findExploration(EXPLORATION_TITLE);
     libraryPage.playExploration(EXPLORATION_TITLE);
 
-    explorationPlayerPage.submitAnswer('TextInput', 'One');
-    explorationPlayerPage.clickThroughToNextCard();
     explorationPlayerPage.submitAnswer('TextInput', 'WrongAnswer1');
-    browser.sleep(3000);
+    explorationPlayerPage.expectLatestFeedbackToMatch(
+      forms.toRichText('Try again'));
     explorationPlayerPage.submitAnswer('TextInput', 'WrongAnswer2');
-    browser.sleep(3000);
+    explorationPlayerPage.expectLatestFeedbackToMatch(
+      forms.toRichText('Try again'));
     explorationPlayerPage.submitAnswer('TextInput', 'WrongAnswer3');
-    browser.sleep(3000);
-    explorationPlayerPage.submitAnswer('TextInput', 'WrongAnswer4');
-    browser.sleep(3000);
-    explorationPlayerPage.submitAnswer('TextInput', 'WrongAnswer5');
-    browser.sleep(3000);
+    explorationPlayerPage.expectLatestFeedbackToMatch(
+      forms.toRichText('Try again'));
     explorationPlayerPage.expectExplorationToNotBeOver();
 
     oppiaLogo.click();
     general.acceptAlert();
     users.logout();
 
     users.login('user1@ExplorationIssues.com');
-    libraryPage.get();
-    libraryPage.findExploration(EXPLORATION_TITLE);
-    libraryPage.playExploration(EXPLORATION_TITLE);
-    general.moveToEditor();
+    creatorDashboardPage.get();
+    creatorDashboardPage.editExploration(EXPLORATION_TITLE);
     explorationEditorPage.navigateToStatsTab();
-
     explorationEditorStatsTab.clickIssue(0, 'Issue 1');
     explorationEditorStatsTab.expectIssueTitleToBe(
-      'Several learners submitted answers to card "Two" several times, ' +
+      'Several learners submitted answers to card "One" 3 times, ' +
```

Comment:
Change this in accordance to the previous comment about the issue statement.

Reply:
Done!